src/render/BpmnPropertiesPanelRenderer.js

```diff
@@ -158,6 +163,11 @@ export default class BpmnPropertiesPanelRenderer {
     return event.providers;
   }
 
+  _restoreCanvasFocus() {
+    const canvas = this._injector.get('canvas');
+    canvas && canvas.restoreFocus && canvas.restoreFocus();
```

Comment:
```suggestion
    // support diagram-js@15+
    if (canvas.restoreFocus) {
      canvas.restoreFocus();
    }
```

Reply:
For a moment I thought we could simply hook into our `focus*` events, until I realized we [don't have those](https://github.com/search?q=repo%3Acamunda%2Fcamunda-modeler+addEventListener%28%27focusin%27&type=code)...